test/e2e/webdriver/driver.js

```diff
@@ -65,6 +65,7 @@ class Driver {
     this.timeout = timeout;
     this.exceptions = [];
     this.errors = [];
+    this.events = [];
```

Comment:
what does this do? Does it ever get referenced?

Reply:
good catch, this was leftover from another implementation that will now be tracked [here](https://github.com/MetaMask/metamask-extension/issues/19385). Removing.